framework/framework.go

```diff
@@ -26,29 +26,29 @@ type Config struct {
 	// within the context initializer function. InitCtxFunc receives the custom
 	// object being reconciled as second argument. Information provided by the
 	// custom object can be used to initialize the context.
-	InitCtxFunc func(ctx context.Context, obj interface{}) (context.Context, error)
-	Logger      micrologger.Logger
-	Resources   []Resource
+	InitCtxFunc    func(ctx context.Context, obj interface{}) (context.Context, error)
+	Logger         micrologger.Logger
+	ResourceRouter func(ctx context.Context, obj interface{}) ([]Resource, error)
```

Comment:
I think it would be nice to add some godoc what is the purpose of the router. It took me some time to find the information below, and I believe it will be more difficult for a newcomer (which is me after 4 weeks :D).

> A resource router got implemented in operatorkit. This will be used to execute different resources in operators based on received custom objects and their versions.

Reply:
Good shout, done.

Reply:
Nice.